lib/ometajs/core.js

```diff
@@ -443,7 +449,10 @@
 		_disablePrependingInput: function() {
 			this._applyWithArgs = function(rule) {
 				var origInput = this.input,
-					ruleArgs = arguments.length > 1 ? __slice.call(arguments, 1) : [];
+					ruleArgs = new Array(arguments.length - 1);
```

Comment:
This function always has to be called with a minimum of 1 argument, `rule`, if it's not it will throw at the `this[rule].apply` point below anyway